README.md

```diff
@@ -4,15 +4,15 @@ Prototype for a new Scrum Poker
 
 ## Development
 
-You need a running [docker](https://docs.docker.com/get-docker/) demon for local development. To test and develop backend and frontend, run
+You need a running [docker](https://docs.docker.com/get-docker/) daemon for local development. To test and develop backend and frontend, run
 
 ```shell
 npm start
 ```
 
 This will spin up the database via `docker` and launch the local API gateway.
 
-In parallel, it will use [Vite](https://vitejs.dev) to serve your app on [http://localhost:5173](http://localhost:5173). Vite will automatically update the browser via hot module reloading on save while providing extremely fast rebuilds.
+In parallel, it will use [Vite](https://vitejs.dev) to serve your app on [http://localhost:5173](http://localhost:5175). Vite will automatically update the browser via hot module reloading on save while providing extremely fast rebuilds.
```

Comment:
```suggestion
In parallel, it will use [Vite](https://vitejs.dev) to serve your app on [http://localhost:5175](http://localhost:5175). Vite will automatically update the browser via hot module reloading on save while providing extremely fast rebuilds.
```